### ISSUE TITLE
Flash cookie removal

### ISSUE THREAD

**Issue by Joel889** [Lusito/forget-me-not#35]
Perhaps for future consideration, being able to remove flash cookies would complete this addon (even if it's only on demand or after restarting the browser).

Thank you

**Comment by Lusito** [comment]
I'll have to decline this one. Two reasons for this:
1. It's impossible to do this with web-extensions, since the web-extension API does not offer the possibility to clear flash cookies.
2. Flash is dead and it will be removed from all browsers in the near future, so the need to remove flash cookies will be gone as well (one reason why it's not in the web-extensions API and never will be)

**Comment by Joel889** [comment]
This makes sense, but now I wonder if the following addon isn't working either:
https://addons.mozilla.org/en-US/firefox/addon/clear-flash-cookies/

**Comment by Lusito** [comment]
That extension is using the clear plugin data API, which is already supported by forget me not.. if that clears flash cookies, then this issue is just a wording problem I guess.

**Comment by Lusito** [comment]
Keep in mind tho, that there is no way to apply the rules on plugin data, as the API does not support this.

**Comment by Joel889** [comment]
I know you said that you probably won't include this, but if anybody is interested the following extension **does** clear flash cookies on startup:
https://addons.mozilla.org/en-US/firefox/addon/clear-flash-cookies/

Tested on FF 58.02, fresh install. When turning the plugin off and restarting the browser, flash cookies remained. When I turned on the plugin and restarted flash cookies were deleted. I tried this several times with either setting.
